src/sql/plot.py

```diff
@@ -501,8 +503,13 @@ def _histogram(table, column, bins, with_=None, conn=None, facet=None):
     # FIXME: we're computing all the with elements twice
     min_, max_ = _min_max(conn, table, column, with_=with_, use_backticks=use_backticks)
 
-    filter_query = f"WHERE {facet['key']} == '{facet['value']}'" if facet else ""
-
+    # Filter query 2 logic based on filter query 1
+    filter_query_1 = f"WHERE {facet['key']} == '{facet['value']}'" if facet else ""
+    filter_query_2 = (
+        f'WHERE "{column}" IS NOT NULL'
```

Comment:
please add test case for both the cases:
1. filter_query_1 != ""
2. filter_query_1 == ""

Reply:
testiing `filter_query_1 == ""` case in test-magic-plot.py
testiing `filter_query_1 != ""` case in test-ggplot.py (where a facet is given)